pkgs/tools/admin/manta/default.nix

```
@@ -1,41 +1,35 @@
 { lib
 , buildNpmPackage
-, fetchurl
+, fetchFromGitHub
 , nodejs
 , installShellFiles
 , testers
 , node-manta
 }:
 
-let
-  source = lib.importJSON ./source.json;
-in
 buildNpmPackage rec {
-  pname = "manta";
-  inherit (source) version;
-
-  src = fetchurl {
-    url = "https://registry.npmjs.org/${pname}/-/${source.filename}";
-    hash = source.integrity;
+  pname = "node-manta";
```

```suggestion
  pname = "manta";
```
Pnames don't need to be unique.

---

With `pname = "manta"`, borg skipped `passthru.tests.version`. Do you know how to keep `pname = "manta"` and tell ofborg to use `node-manta`?

---

@teutat3s Can you try to change the pname just so we can see the ofborg failure?

---

The issue is probably that `manta` is an alias to `pkgs.node-manta`, and ofborg skips aliases: https://github.com/NixOS/nixpkgs/blob/479646999c8cf4f7580d294b36742060fe1de170/pkgs/development/node-packages/aliases.nix#L91

---

That, or it's related to the fact that there is a `manta` package already: https://github.com/NixOS/nixpkgs/blob/479646999c8cf4f7580d294b36742060fe1de170/pkgs/top-level/all-packages.nix#L39283

So ofborg noticed that `manta` changed and tried to build `manta`, not `node-manta`, as this package is defined as: https://github.com/NixOS/nixpkgs/blob/479646999c8cf4f7580d294b36742060fe1de170/pkgs/top-level/all-packages.nix#L14063

---

But `pkgs.manta` didn't change, only `pkgs.node-manta` did. And that is the package mentioned in the commit message.
@teutat3s You have to use `node-manta` in the commit message regardless of what the pname is.

---

Thank you, that makes sense. I first had `manta` in the commit message, that of course confused borg.